src/main/java/com/example/ourPick/controller/ItemController.java

```diff
@@ -18,5 +21,11 @@ public void regItem(@RequestBody ItemRequestRequest item){
         itemService.regItem(item);
     }
 
+    @GetMapping
+    public List<ItemResponse> getItems(){
+        List<ItemResponse> resultList = itemService.getItems();
+        return resultList;
```

Comment:
@dbchae22 
사소하지만 List라는 네이밍 보단 복수명 s를 쓰는게 더 좋아요~!
```suggestion
        List<ItemResponse> results = itemService.getItems();
        return results;
```